readme.md

```diff
@@ -822,6 +822,21 @@ console.log(stdout);
 
 When combining `inherit` with other values, please note that the child process will not be an interactive TTY, even if the parent process is one.
 
+### Redirect a Node.js stream from/to stdin/stdout/stderr
+
+When passing a Node.js stream to the [`stdin`](#stdin), [`stdout`](#stdout-1) or [`stderr`](#stderr-1) option, Node.js requires that stream to have an underlying file or socket, such as the streams created by the `fs`, `net` or `http` core modules. Otherwise the following error is thrown.
+
+```
+TypeError [ERR_INVALID_ARG_VALUE]: The argument 'stdio' is invalid.
+```
+
+This limitation can be worked around by passing `[nodeStream, 'pipe']` instead of `nodeStream`.
+
+```diff
+- await execa(..., { stdout: nodeStream })
++ await execa(..., { stdout: [nodeStream, 'pipe'] })
+```
```

Comment:
Maybe also mention that they can just pass a web stream in instead?

Reply:
Done :+1: